catalog/camel-report-maven-plugin/src/main/java/org/apache/camel/maven/RouteCoverageMojo.java

```diff
@@ -503,9 +503,7 @@ private static void gatherRouteCoverageSummary(
 
     private static String padString(int level) {
         StringBuilder sb = new StringBuilder();
-        for (int i = 0; i < level; i++) {
-            sb.append("  ");
-        }
+        sb.append("  ".repeat(Math.max(0, level)));
```

Comment:
Should it not just be .repeat(level). The level is not intended to be negative

Reply:
it should. And the StringBuilder isn't necessary after the loop gor removed. So it will be at least one more commit here.
It was just the automated substitution - to be on the safe side

Reply:
Can you remove the Math.max code as repeat already check that the input must be a positive value

Reply:
fixed